src/layouts/ModelizerDrawLayout.vue

```diff
@@ -92,6 +112,19 @@ async function initView() {
 onMounted(() => {
   initView();
 });
+
+onUnmounted(() => {
```

Comment:
Why do you `unsubscribe` if no `subscribe` is done ?
I think `pluginInitSubscription` is useless.

Reply:
I don't remember having written this code. Must have been placed here by mistake.

I removed it.